src/lib/transition_handler/catchup_scheduler.ml

```diff
@@ -182,7 +182,7 @@ module Make (Inputs : Inputs.S) = struct
               ; ( "cached_transition"
                 , With_hash.data transition_with_hash
                   |> Inputs.External_transition.to_yojson ) ]
-            "timed out waiting for the parent of $cached_transition after \
+            "Timed out waiting for the parent of $cached_transition after \
```

Comment:
change this to debug

Reply:
We should add an `info` message that we're starting catchup if there isn't one already.